admin/gv_queue.php

```diff
@@ -200,6 +201,9 @@ function init()
 //      $contents[] = array('align' => 'center', 'text' => '<a href="' . zen_href_link('gv_queue.php', 'action=confirmrelease&gid=' . $gInfo->unique_id . '&page=' . $_GET['page'],'NONSSL') . '">' . zen_image_button('button_confirm_red.gif', IMAGE_CONFIRM) . '</a> <a href="' . zen_href_link('gv_queue.php', 'action=cancel&gid=' . $gInfo->unique_id . '&page=' . $_GET['page'],'NONSSL') . '">' . zen_image_button('button_cancel.gif', IMAGE_CANCEL) . '</a>');
       break;
     default:
+      if (!isset($gInfo) || !is_object($gInfo)) { 
+        $gInfo = new objectInfo(array());

```

Comment:
Thought this was supposed to rid notices, won't a notice be generated 2 lines down if this is executed? (no `unique_id`, no `date_created`, etc...)